### ISSUE TITLE
Unable to start Admin UI, 'ambiguous indirect export: default'

### ISSUE THREAD

**Issue by mackcheesman** [exogee-technology/graphweaver#290]
**Describe the bug**
When I try to use the Admin UI, nothing loads - I get an error in the console: `Uncaught SyntaxError: ambiguous indirect export: default`
This persists across any attempts to rerun pnpm build & pnpm start.

**To Reproduce**

1. Create a local MySQL database using the scripts given in the [adding local authentication](https://graphweaver.com/docs/adding-local-authentication) example.
2. Use `npx graphweaver@latest init` to create a new project & `npx graphweaver@latest import mysql` to create a basic backend
3. Run `pnpm build && pnpm start`
4. Access the admin UI at `http://localhost:9000/`
5. Open the browser console

**Expected behavior**
Previously, the Admin UI would load without issue

**Actual behavior**
Nothing appears - the page remains a blank dark purple.
The error in the console reads `Uncaught SyntaxError: ambiguous indirect export: default`, and gets thrown at `formik.esm.js:8:8`
line 8:
```
import hoistNonReactStatics from "/@fs/path/to/my/project/node_modules/.pnpm/hoist-non-react-statics@3.3.2/node_modules/hoist-non-react-statics/dist/hoist-non-react-statics.cjs.js?v=4d7a8c23";
```

**Screenshots or Screencasts**
If applicable, add screenshots or screencasts to help explain the issue.

**Desktop (please complete the following information):**

- OS and version: OS and version: MacOS 14.2.1 (M1)

**Comment by hunthunt2010** [comment]
@agaw is there any ETA on your PR being merged?

**Comment by thekevinbrown** [comment]
Hey @hunthunt2010, we're doing two things to resolve this:

- Fix the actual error by merging this PR.
- Updating our end to end tests to test the built version of the packages so that if a similar type of error crops up we catch it in CI.

@steve-keep is working on this now and we expect to have a resolution in the next few days.

**Comment by steve-keep** [comment]
Hey @hunthunt2010 This is now resolved in 0.2.18. Please let us know if you still have an issue.

Thanks!

**Comment by steve-keep** [comment]
Resolved how we were bundling the admin UI components.
